src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,105 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75d;
+    private int threshold;
+    private int capacity;
+    private int size;
+    private Node<K, V>[] buckets;
+
+    public static class Node<K, V> {
```

Comment:
```suggestion
    private static class Node<K, V> {
```

I think we could make it private